docs/reference/cluster/remote-info.asciidoc

```diff
@@ -67,3 +67,9 @@ Refer to <<skip-unavailable-clusters>>.
 `max_proxy_socket_connections`::
     The maximum number of socket connections to the remote cluster when proxy
     mode is configured.
+
+`cluster_credentials`::
+// TODO: fix the link to new page of API key based remote clusters
```

Comment:
This PR has quite a few of this TODO which can be fixed once #98330 is in place.